spring-mvc/pom.xml

```diff
@@ -39,11 +39,19 @@
     </build>
 
     <dependencies>
+        <dependency>
+            <groupId>com.kodemon</groupId>
+            <artifactId>sample-data</artifactId>
+            <version>0.9-SNAPSHOT</version>
```

Comment:
Here, pls use notation from other modules.
`<version>${project.parent.version}</version>` or something like that.
This approach (that I suggested) is (I think) simplier, because it ensures that correct version are being used